docs/source/tutor/_hello_world.ipynb

```diff
@@ -0,0 +1,698 @@
+    "\n",
+    "- A **method** is a function that is defined in a **class** and can be called on an **object** of that class. For example, the \"bark\" method defined in the `Dog` class can be called on the object \"my_dog\" to make it bark.\n",
+    "</details>"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Follow these steps to load the `bluesky` and `databroker` packages for data collection activities."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "1"
+      ]
+     },
+     "execution_count": 1,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "import pathlib, sys, os\n",
+    "my_path = str(pathlib.Path.home() / \"bluesky\")\n",
+    "sys.path.append(my_path)\n",
+    "\n",
+    "import bluesky\n",
+    "import databroker\n",
+    "from bluesky.callbacks.best_effort import BestEffortCallback\n",
+    "\n",
+    "cat = databroker.temp().v2\n",
+    "RE = bluesky.RunEngine()\n",
+    "RE.subscribe(cat.v1.insert)\n",
+    "RE.subscribe(BestEffortCallback())"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<details>\n",
+    "<summary>Line by line explanation</summary>\n",
+    "\n",
+    "- `my_path = ...` define a variable `my_path` as the \"bluesky\" directory located in the user's home directory\n",
+    "- `sys.path.append(my_path)` adds this directory to the list of directories that Python will search when you try to import a module.\n",
+    "- `cat = databroker.temp().v2` creates a temporary databroker instance and assigns it to the variable `cat`. This allows you to interact with the databroker instance and access its functionality, such as querying for data, creating runs, or accessing metadata.\n",
+    "- `RE = bluesky.RunEngine()` creates a new instance of the RunEngine class and assigns it to the variable `RE`. The RunEngine is the central component of Bluesky, responsible for managing the execution of experimental plans and coordinating communication between other components, such as detectors and data acquisition devices.\n",
+    "- `RE.subscribe(cat.v1.insert)` sets up a subscription to the RunEngine instance (`RE`) that will insert data into a databroker instance (`cat.v1`) when each run is complete: the data collected during each run will automatically be inserted into the databroker instance at the end of the run. This ensures that the data is stored and available for later analysis and processing.\n",
+    "- `RE.subscribe(BestEffortCallback())` sets up a subscription to the RunEngine instance (`RE`) that will automatically print a summary of the run data at the end of each run.\n",
```

Comment:
change `print a summary of the run data at the end of each run.` to `show a chart and print a table of the run data and a summary at the end of the run.`